### ISSUE TITLE
Failed on xhr.open('GET', global.location.host ? '/' : 'https://example.com')

### ISSUE THREAD

**Issue by astappev** [jhiesey/stream-http#53]
Hello,

I tried to use browserified package which used http in Firefox extension, and I have got exception on loading script caused by lines
```
var xhr = new global.XMLHttpRequest()
// If location.host is empty, e.g. if this page/worker was loaded
// from a Blob, then use example.com to avoid an error
xhr.open('GET', global.location.host ? '/' : 'https://example.com')
```
from [capability.js](https://github.com/jhiesey/stream-http/blob/master/lib/capability.js#L9)

The error was because my extension loaded  on the page chrome://seleniumbuilder3 and global.location.host variable was equal to "seleniumbuilder3".
<details>
  <summary>My global.location variable</summary>
  ![2016-09-16_170013](https://cloud.githubusercontent.com/assets/4512729/18590495/5a9072d2-7c2f-11e6-9032-c0f839a428e2.png)
</details>

I simplest replaced line above to `xhr.open('GET', 'https://example.com')` and it works.

So, I propose to add checking if global.location.protocol is not "chrome:" or something else (I don't have so many experience with it). It can save a couple of hours for another devs who will try to use it in their extensions.

**Comment by jhiesey** [comment]
Thanks for reporting this. I'll be sure to fix this soon!

**Comment by jhiesey** [comment]
Fixed properly in 4e7103a5a985a0d75ecb97be46020246de426c3f
